### PR TITLE
Remove outdated reference in Lisk docs index page

### LINKED ISSUE #1918: Remove outdated reference in Lisk docs index page
### Description
 Remove outdated reference to snap page in the Index page

#### Affected branch / URL
`development`

#### Page location
`run-blockchain/index.adoc`

### PR DESCRIPTION
**Pull request by chris529**
Closes #1918

### CODE REVIEW

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 1]
```diff
@@ -18,7 +18,7 @@ Muhammad Talha <muhammad.talha@lightcurve.io>
 :url_core: {docs_core}index.adoc
 :url_core_setup_npm: {docs_core}setup/npm.adoc
 :url_core_setup_binary: {docs_core}setup/binary.adoc
-:url_core_setup_snap: {docs_core}setup/snap.adoc
+// :url_core_setup_snap: {docs_core}setup/snap.adoc
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]:
```suggestion
```

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 2]
```diff
@@ -227,7 +227,7 @@ To set up Lisk Core, choose a distribution and follow the respective setup guide
 
 * *NPM:* xref:{url_core_setup_npm}[]
 * *Binary:* xref:{url_core_setup_binary}[]
-* *Snap:* xref:{url_core_setup_snap}[]
+// * *Snap:* xref:{url_core_setup_snap}[]
```
Comment (TalhaMaliktz) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]:
```suggestion
```

**Review by TalhaMaliktz — CHANGES_REQUESTED** [review]
No need to keep them, simply remove these lines.

**Review by TalhaMaliktz — APPROVED** [review]
LGTM

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 3]
```diff
@@ -18,7 +18,6 @@ Muhammad Talha <muhammad.talha@lightcurve.io>
 :url_core: {docs_core}index.adoc
 :url_core_setup_npm: {docs_core}setup/npm.adoc
 :url_core_setup_binary: {docs_core}setup/binary.adoc
-:url_core_setup_snap: {docs_core}setup/snap.adoc
 // :url_core_setup_docker: {docs_core}setup/docker.adoc
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]:
```suggestion
:url_core_setup_docker: {docs_core}setup/docker.adoc
```

File: docs/modules/ROOT/pages/run-blockchain/index.adoc [hunk 4]
```diff
@@ -227,7 +226,6 @@ To set up Lisk Core, choose a distribution and follow the respective setup guide
 
 * *NPM:* xref:{url_core_setup_npm}[]
 * *Binary:* xref:{url_core_setup_binary}[]
-* *Snap:* xref:{url_core_setup_snap}[]
 // * *Docker:* xref:{url_core_setup_docker}[]
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/run-blockchain/index.adoc]:
```suggestion
* *Docker:* xref:{url_core_setup_docker}[]
```

**Review by Tschakki — APPROVED** [review]
LGTM, just added suggestions to add the docker links back in, as we have updated the corresponding guide now.

File: docs/modules/ROOT/pages/run-blockchain/index.adoc (hunk 3 above)
Comment (chris529) [reply docs/modules/ROOT/pages/run-blockchain/index.adoc]: Good point Mona! Now done 👍

**Review by TalhaMaliktz — APPROVED** [review]

**Pull request merged**